aspnetcore/signalr/groups.md

```diff
@@ -26,17 +26,10 @@ Send a message to a specific user by passing the user identifier to the `User` f
 > [!NOTE]
 > The user identifier is case-sensitive.
 
-```csharp
-public Task SendPrivateMessage(string user, string message)
-{
-    return Clients.User(user).SendAsync("ReceiveMessage", message);
-}
-```
+[!code-csharp[Configure service](groups/sample/hubs/chathub.cs?range=29-32)]
 
 The user identifier can be customized by creating an `IUserIdProvider`, and registering it in `ConfigureServices`.
```

Comment:
If we do that I think this line and the one below should be removed too otherwise it looks odd.

Reply:
SGTM will do